bagpipes/fitting/fitted_model.py

```
@@ -240,6 +255,18 @@ def _lnlike_spec(self):
 
             return K_spec - 0.5*self.chisq_spec
 
+    def _lnlike_line_fluxes(self):
+        """ Calculates the log-likelihood for spectral indices. """
+
+        labels = self.galaxy.line_labels
+        model_line_fluxes = [self.model_galaxy.line_fluxes[l] for l in labels]
```

Ruff (0.14.5)

Ambiguous variable name: `l`

(E741)